test/e2e/actuals_test.go

```diff
@@ -82,6 +82,23 @@ func workNamespaceAndConfigMapPlacedOnClusterActual(cluster *framework.Cluster)
 	}
 }
 
+func workNamespacePlacedOnClusterActual(cluster *framework.Cluster) func() error {
+	workNamespaceName := fmt.Sprintf(workNamespaceNameTemplate, GinkgoParallelProcess())
+
+	return func() error {
+		return validateWorkNamespaceOnCluster(cluster, types.NamespacedName{Name: workNamespaceName})
+	}
+}
+
+func workConfigMapPlacedOnClusterActual(cluster *framework.Cluster) func() error {
```

Comment:
what's the diff between this func and func workNamespaceAndConfigMapPlacedOnClusterActual?

They're the same.